Description: Fix a log level check to not be always true.
Forwarded: not yet
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2015-04-27

--- a/src/log.c
+++ b/src/log.c
@@ -215,7 +215,7 @@
                 file_putline(outfile, line); /* send log to file */
         }
     } else if(mode==LOG_MODE_ERROR) {
-        if(level>=0 || level<=7) /* just in case */
+        if(level>=0 && level<=7) /* just in case */
             line=str_printf("[%c] %s", "***!:.  "[level], text);
         else
             line=str_printf("[?] %s", text);
